### PR TITLE
add coverage for cli command, update readme

### PR DESCRIPTION
**Pull request by timofei-iatsenko**

### CODE REVIEW

**Comment by timofei-iatsenko** [comment]
Coverage report is failing because coverage is actually low. The `cli command` wasn't covered at all,
i've added a basic tests for it. but it doesn't cover all 100% cases.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ jobs:
         run: npm run lint:types
 
       - name: Test
-        run: npm test -- --coverage
+        run: npm test -- --coverage --runInBand
```
Comment (timofei-iatsenko) [inline .github/workflows/test.yml:35]: for small amount of tests spawning threads is longer than just run everything in band.

File: Readme.md [hunk 2]
```diff
@@ -30,12 +35,12 @@ _Pseudolocale_ includes a command line interface to make it easy to incorporate
 Note: Nodejs must be installed to use the command line interface.
 
 ```bash
-./dist/cli.mjs --string 'This is going to be pseudolocalized %token%.'
+pseudolocale --string 'This is going to be pseudolocalized %token%.'
```
Comment (timofei-iatsenko) [inline Readme.md:38]: you don't need to call `cli.js` directly. Thanks to `bin` field in `package.json` your package manager make it available as a command for you.

File: jest.config.js [hunk 3]
```diff
@@ -4,4 +4,5 @@
 module.exports = {
   preset: 'ts-jest',
   testEnvironment: 'node',
+  collectCoverageFrom: ['./src/**/*.{ts,tsx}'],
```
Comment (timofei-iatsenko) [inline jest.config.js:7]: Regularly, coverage collected only from **tested** files. It means that if you have 100 files, and only one is used in test you will have 100% coverage, despite other 99 files don't tested at all. This adds collecting coverage from files which were not imported in tests.

**Review by MartinCerny-awin — APPROVED** [review]

**Pull request merged**